Gateway/Http/Client/ClientService.php

```diff
@@ -101,10 +112,13 @@ public function placeRequest(TransferInterface $transferObject)
             $rawResponse = $response->getRawBody();
             $response = $this->converter->convert($rawResponse);
         } catch (\Exception $e) {
+            $log['error'] = $e->getMessage();
+            $this->logger->error($log);
             throw new ClientException(__($e->getMessage()));
         } finally {
             $log['response'] = $response;
             $this->logger->debug($log);
+            $this->affirmLogger->debug('Astound\Affirm\Gateway\Http\Client\ClientService::placeRequest', $log);
```

Comment:
can we also log the response if possible

Reply:
The block within `finally`will run whether or not an exception occurred so response object will still be logged there in case of exception.